Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.yml

```diff
@@ -407,8 +407,8 @@ script:
     deprecated: false
     description: Searches for sessions in AutoFocus. To view results, run the autofocus-sessions-search-results
       command with the returned AF Cookie. The AF Cookie expires 120 seconds after
-      the search completes. Note that due to an API limitation, passing more than 100 values to an IOC argument (e.g. 100 IPs) might cause a failure. In this case, you will need to iterate over the IPs and run the autofocus-sessions-search-results command for each AF cookie.
-      `Autofocus Query Samples, Sessions and Tags` Playbook is recommended for querying and polling.
+      the search completes. The `polling` argument was added in XSOAR 6.2.0. It enables to handle the search in a single command,
+      foregoing the need to run the `Autofocus Query Samples, Sessions and Tags` Playbook.
```

Comment:
```suggestion
      the search completes. The `polling` argument was added in XSOAR 6.2.0. It enables handling the search in a single command,
      rather than using the `Autofocus Query Samples, Sessions and Tags` Playbook.
```